### DISCUSSION TITLE
podman installation on centos8 arm64

### DISCUSSION THREAD

**Discussion by kevinl2022** [containers/podman#13887] [General]
wonder if you have any podman bin package provided to run on centos8 arm64 directly ? 
I try to compiled it on my centos8 arm64 with the podman source package, but it does't make any progress in two days( lots of dependency/problem to troubleshoot )

Thanks!

**Comment by baude** [comment]
we do not offer such a thing upstream.

**Comment by kevinl2022** [comment]
> we do not offer such a thing upstream.

thanks for thr reply. could you provide any info about installation on arm? Appreciate for any info!

**Comment by rhatdan** [comment]
This is a discussion not an issue.

**Reply by kevinl2022** [reply]
OK, thank you
